game.js

```diff
@@ -0,0 +1,40 @@
+class Game {
+
+  constructor() {
+    this.rolls = [];
```

Comment:
perhaps could refactor to have a frame class? currently no concept of a frame, only rolls. having a frame class/frame objects could allow you to build onto the logic going forward (even if its not CLI)